com.ibm.streamsx.topology/opt/python/packages/streamsx/topology/graph.py

```diff
@@ -188,3 +191,44 @@ def getSPLOutputPort(self):
         _oport["name"] = self.name
         _oport["connections"] = [port.name for port in self.inputPorts]
         return _oport
+
+class SPLMarker(object):
```

Comment:
Technically this is not an SPLMarker, since it's not an SPL artifact. maybe just Marker